### PR TITLE
task done

### PR DESCRIPTION
**Pull request by maksym-mishchanchuk**

### CODE REVIEW

**Review by GitRoko — COMMENTED** [review]
Нужно значения вынести в переменные

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount;
+
+  if (days < 3) {
+    amount = (days * 40);
```
[the remark this answers is not in the document]
Comment (maksym-mishchanchuk) [reply src/calculateRentalCost.js]: поправив)

**Review by GitRoko — APPROVED** [review]

File: src/calculateRentalCost.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (GitRoko) [reply src/calculateRentalCost.js]: Approve!

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -28,7 +28,24 @@
...
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount;
+  const priceForDays = 40;
+  const saleDiscountUp7Days = 20;
+  const saleDiscountAfter7Days = 50;
+
+  if (days < 3) {
+    amount = (days * priceForDays);
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:37]: Don't repeat yourself: create variable for days * priceForDays value

Comment (Anna-Tkachenko) [reply src/calculateRentalCost.js:37]: You can also set one of values by default, one of this if statements is redundant here

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**